.github/workflows/build_all.yml

```diff
@@ -145,10 +145,13 @@ jobs:
 
   MacOS:
     if: github.event_name != 'schedule' || github.repository == 'mesonbuild/wrapdb'
-    runs-on: macos-latest
+    runs-on: ${{ matrix.platform == 'arm64' && 'macos-14' || 'macos-12' }}
```

Comment:
hmm? Would be simpler to use 14 unconditionally.

Reply:
AFAICT there's no x86_64 for 14.